src/installing-workflow/configuring-registry.md

```diff
@@ -17,7 +17,7 @@ When using a private registry the docker images are no longer pulled by Deis Wor
 Deis Workflow currently supports
   1. Google Container Registry([gcr][gcr]).
   2. EC2 Container Registry([ecr][ecr]).
-  3. off-cluster storage providers like dockerhub, quay.io, etc., or self hosted docker registry.
+  3. off-cluster : Any provider which provides long lived auth(username/password) like [Azure Container Registry][acr], [dockerhub][dockerhub], [quay.io][quay], etc., or self hosted docker registry.
```

Comment:
minor grammatical errors (spacing):

"3. off-cluster: Any provider which provides long lived authentication (username/password) like [Azure Container Registry][acr], [dockerhub][dockerhub], [quay.io][quay], etc., or self hosted docker registry."

I'd also prefer if we expand "auth" to "authentication" as auth can be one of two things: authentication or authorization. It's not completely clear at first glance.